learn/on-premise/azure/install/initilize-setup.md

```diff
@@ -19,3 +19,13 @@ sidebar_label: "Initialize Setup"
 - Go to  <http://Platform-Instance-IP:8080> as directed by the above command output to do the WME setup.
 
     [![setup initialization](/learn/assets/wme-setup/wavemaker-setup-initialization.jpg)](/learn/assets/wme-setup/wavemaker-setup-initialization.jpg)
+
+- From WME version 10.5.1 version users can use their own user for communication between Platform, Studio Workspace and Appdeployment Instances.for use custom user for communication and operations do [prerequisites](../install-prerequisites.md) as per given steps.
```

Comment:
No need to mention version.  
the doc always refers to lateste version so that is 10.5.3.